client/App.vue

```diff
@@ -40,6 +41,11 @@ export default {
         type: '',
         timeout: undefined,
       },
+      announcement: {
+        message: '🪐 newer version is avaialable',
+        show: false,
+        link: ''
```

Comment:
Is there a link that makes sense (to the release notes of new version maybe)?

Reply:
this is just a placeholder, another PR that implements version-check will pass the link

Reply:
+ for the github link